jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/resources/org/jbpm/workbench/es/client/i18n/Constants.properties

```diff
@@ -105,4 +100,21 @@ UnacknowledgedErrors=Unackowledged Execution Errors
 AcknowledgedErrors=Acknowledged Execution Errors
 ViewProcessInstance=View process
 ViewTask=View task
-RequestRequeued=Request {0} put back into execution queue
\ No newline at end of file
+RequestRequeued=Request {0} put back into execution queue
+Cancel_Job=Cancel job
+Cancel_Job_Sure=Are you sure about to cancel this Job?
```

Comment:
Why was this changed? It sounded better to me how it was i.e. ```Are you sure you want to cancel this Job?```
The same comment about the new bulk messages.

Also, one minor thing regarding the capitalization. When we are asking for confirmation about one job, ```J``` in ```Job``` is a capital letter, and when there's a bulk of them ```j``` is a small letter in ```jobs```